UnchainedLauncher.GUI/App.xaml.cs

```diff
@@ -135,16 +146,20 @@ select chiv2Installations.Path
             );
 
             var unchainedLauncher = new UnchainedChivalry2Launcher(
-                unchainedContentPreparer.Sub(sigLaunchPreparer),
+                unchainedContentPreparer.Sub(sigLaunchPreparer, _ => Unit.Default),
                 unchainedProcessLauncher,
                 Directory.GetCurrentDirectory(),
+#if DEBUG_FAKECHIVALRYLAUNCH
+                // point to a directory that won't actually have dlls. That way, we
+                // don't shove dlls into processes that don't need them and cause crashes
+                new DllInjector(Path.Combine(Directory.GetCurrentDirectory(), FilePaths.PluginDir+"/emptydir"))
```

Comment:
`DllInjector` now implements `IProcessInjector`, and classes depend on the `IProcessInjector` instead.  This could be updated to point to an `IProcessInjector` which performs a noop